docs/tutorials/python-api.md

```diff
@@ -117,9 +113,8 @@ rag =  Rag(config)
 ```
 
 !!! note
-
```

Comment:
Same as above.

Reply:
[added back](https://github.com/Quansight/ragna/pull/196/commits/576d96eb59c66245ac8611177e9d74a842b317c9) the new line